packages/lib/RotatingLogs.ts

```diff
@@ -29,7 +29,8 @@ export default class RotatingLogs {
 		const files: Stat[] = await this.fsDriver().readDirStats(this.logFilesDir);
 		for (const file of files) {
 			if (!file.path.match(/^log-[0-9]+.txt$/gi)) continue;
-			const ageOfTheFile: number = Date.now() - file.birthtime;
+			const timestamp: string = file.path.match(/[0-9]+/g)[0];
+			const ageOfTheFile: number = Date.now() - timestamp;
```

Comment:
For some reason tests were failing when using `file.mtime` (the modification time from stats), so I decided to use the timestamp from the name of the file. I believe its the best solution possible as the timestamp from the name of the file always is going to have the correct timestamp (since when was it renamed). Now, if someone change to file.birthtime tests will fail.

Reply:
Yes thats' a good change, then you don't need to worry about the unreliable filesystem timestamps

Reply:
That line is not valid typescript - see CI

Reply:
Yes, I thought it's better this way.
Ok, I'll fix that. Thanks!